tests/test_vizplugin.py

```diff
@@ -27,12 +29,11 @@ def f(data):
         if m_type == "event" and payload["when"] == "pre-save":
             return {
                 "action": "handle_data",
-                "handler": f
+                "handler": f,
             }
 
-        if m_type == "command":
-            if payload["cmd_type"] == "terminate":
-                return {"success": self.terminate_well}
+        if m_type == "command" and payload["cmd_type"] == "terminate":
```

Comment:
Please revert this change - this is the structure left for future development. Currently there is only one command `terminate`, but it's possible that there are more in the future.